### ISSUE TITLE
did autokeras support. multi-gpu for train?

### ISSUE THREAD

**Issue by bestpredicts** [keras-team/autokeras#1006]
Hi, I  have a  server with two gpus , I use autokeras  for  training,  but it seems only one gpu is working and the other does not work.

**Comment by bestpredicts** [comment]
here is my code `#%%
import os
import math
import codecs
import random
import numpy as np
from glob import glob
from PIL import Image
import cv2
from sklearn.model_selection import train_test_split
from keras.utils import to_categorical
from tqdm import tqdm
import autokeras as ak
import time
import os
os.environ["CUDA_VISIBLE_DEVICES"] = "0,1"


DEBUG = True
PRINT = False
img_width = 50
img_height = 50




def data_flow(train_data_dir):  # need modify
    label_files = glob(os.path.join(train_data_dir, '*.txt'))
    random.shuffle(label_files)
    img_paths = []
    labels = []
    for index, file_path in enumerate(label_files):
        with codecs.open(file_path, 'r', 'utf-8') as f:
            line = f.readline()
        line_split = line.strip().split(', ')
        if len(line_split) != 2:
            print('%s contain error lable' % os.path.basename(file_path))
            continue
        img_name = line_split[0]
        label = int(line_split[1])
        img_paths.append(os.path.join(train_data_dir, img_name))
        labels.append(label)
    return  img_paths,labels


if DEBUG:
    img_paths,labels = data_flow("/home/dy/autodl/garbage/garbage_classify/train_data")
    if PRINT:
        print(img_paths[:10],labels[:10])




def prepare_data(list_of_images,labels):
    """
    Returns two arrays: 
        x is an array of resized images
        y is an array of labels
    """
    x = [] # images as arrays
    y = [] # labels
    
    for image in tqdm(list_of_images):
        x.append(cv2.resize(cv2.imread(image), (img_width,img_height), interpolation=cv2.INTER_CUBIC))

    y = labels 
    return x, y



if DEBUG:
    X,Y=  prepare_data(img_paths,labels)
    if PRINT:
        print(X)
        print(Y)

    X_train, X_val, Y_train, Y_val = train_test_split(X,Y, test_size=0.2, random_state=1)



X_train = np.array(X_train)
Y_train =  to_categorical(Y_train)

X_val = np.array(X_val)
Y_val = to_categorical(Y_val)

start = time.time()


input_node = ak.ImageInput()
output_node = ak.Normalization()(input_node)
output_node = ak.ImageAugmentation(percentage=0.3)(output_node)
output_node = ak.ResNetBlock(version='v2')(output_node)
output_node = ak.ClassificationHead()(output_node)
clf = ak.AutoModel(inputs=input_node, outputs=output_node, max_trials=10)
clf.fit(X_train, Y_train, epochs=50)
# Evaluate on the testing data.
print('Accuracy: {accuracy}'.format(
    accuracy=clf.evaluate(X_val, Y_val)))

end = time.time()

minute = (end-start)/60


print("总共运行时间 %f min"%minute)


`
